source/chapters/library.rst

```diff
@@ -828,3 +828,80 @@ different location. Select :guilabel:`Use Default Library` to reset.
 
 .. seealso:: External libraries can be disabled under
              :menuselection:`Preferences --> Library`.
+
+Removing tracks from the library
+================================
+
+Removing tracks from the Mixxx library will **not** physically delete them from
+your drive. However it does delete extra metadata Mixxx might have (such as
+hotcues and the beatgrids), and removes links to playlists or crates.
+
+#. Click the :guilabel:`Library` item in the sidebar.
+#. Find and select the tracks you want to remove, perform a right-click on them
+   and select :guilabel:`Hide from Library` from the context menu.
+#. Expand the :guilabel:`Library` item in the sidebar and click on the
+   :guilabel:`Hidden tracks` sub-item. All tracks that were set to be hidden
+   from the library will appear here.
+#. Select the tracks you want to remove, or use the :guilabel:`Select All`
+   button.
+#. To confirm you want to permanently remove these tracks from the library,
+   click :guilabel:`Purge`.
+
+.. hint:: If you later decide to add some of the deleted tracks back, import
+          them to the Mixxx library again, see :ref:`configuration-import`.
+
+.. _getting-started-file-format-compatibility:
+
+Compatible file formats
+=======================
+Mixxx supports a variety of audio file formats:
+
+Lossless:
+
+* :term:`FLAC` (flac)
+* :term:`Wave <WAV>` (wav)
+* :term:`Aiff <AIFF>` (aiff, aif)
+
+Lossy:
+
+* :term:`MP3` (mp3)
+* :term:`Ogg Vorbis` (ogg)
+* :term:`AAC` (aac, m4a, mp4)
+* :term:`Opus` (opus)
+
+Lossless formats preserve every detail of the original recording and are
+recommended for the best sound quality. The FLAC format compresses the
+files to take less space on your computer without sacrificing audio quality.
+WAV and AIFF files are uncompressed and take up much more space.
+
+Lossy formats take up less storage space on your computer than lossless
+formats at the expense of reducing sound quality. However, modern hard drives
```

Comment:
I think saying "if disk space is not an issue" is too vague. I think it is helpful to give the perspective that modern HDs & SSDs can store many thousands of tracks in FLAC.

Reply:
Maybe you can find an other less meaningless sentence. What do you like to express. 
"Use Flac, there should be now excuses, because of missing HD space"?
I have a not that recent external HDD and it is occupied with mp3s. We may also consider that not all users have a high speed internet connection.

You may give a real example: 
A typical USB Hard disk has 1 TB and can store more than 30000 FLAC files.

Reply:
Maybe you can find an other less meaningless sentence. What do you like to express. 
"Use Flac, there should be now excuses, because of missing HD space"?
I have a not that recent external HDD and it is occupied with mp3s. We may also consider that not all users have a high speed internet connection.

You may give a real example: 
A typical USB Hard disk has 1 TB and can store more than 30000 FLAC files.

Reply:
Giving actual numbers is a good idea. A 4 minute 44.1 kHz 16 bit FLAC file takes about 25 MB. A 4 minute 320 kbps CBR MP3 takes about 9.2 MB. Assuming an average track length of about 4 minutes, a 1 TB drive would fit 42000 FLAC files or 115000 MP3s.